packages/api/src/middleware/authentication.ts

```diff
@@ -228,9 +228,25 @@ const extractUserEmailFromAuthToken = async (
   next();
 };
 
+const extractUserIsAdminFromAuthToken = async (
```

Comment:
You asked about unit tests and I think my answer was incomplete. To answer more thoroughly, we favor functional tests for our controllers (and also, so far, for lambda handlers and cron scripts), because we want to ensure we test the SQL against a real database, and because separating the service from the controller in tests doesn't seem to add much value. We do generally unit test code that is shared between many controllers, such as middleware or shared validators.

Reply:
It looks like this function still needs unit tests